modules/cli-logging-in-web.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="cli-logging-in-web_{context}"]
+= Logging in to the OpenShift CLI using a web browser
+
+You can log in to the OpenShift CLI (`oc`) with the help of a web browser to access and manage your cluster. This allows users to avoid inserting their access token into the command line.
+
+== Logging in through a web browser with the CLI
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster.
+* You must have installed the OpenShift CLI (`oc`).
+* You must have a browser installed.
+
+.Procedure
+
+. Enter the `oc login` command with the '--web' flag:
++
+[source,terminal]
+----
+$ oc login <https://api.your-openshift-server.com> --web
+----
+
+[WARNING]
+====
+Logging in through the web browser runs a server on localhost with HTTP, not HTTPS. Use with caution on multi-user workstations.
+====
+
+. The web browser will open automatically. If it does not, click the link in the command output. If you do not specify the OpenShift Container Platform server 'oc' will try to open the OpenShift Console of the cluster specified in the current 'oc' config file. If no 'oc' configuration exists, 'oc' will prompt interactively for the server URL.
++
+.Example output
+
+[source,terminal]
+----
+$ oc login --web <1>
+Opening login URL in the default browser: <https://openshift.example.com>
+Opening in existing browser session.
+----
+<1> If needed, you can specify the callback port.
```

Comment:
I would either mention the server URL here:
```suggestion
<1> If needed, you can specify the callback port and server URL.
```

or omit `localhost:8443` from the example; currently it reads as if the URL is part of the callback port.

Reply:
Change made, thank you!